docs/apis-tools/camunda-api-rest/specifications/add-application-to-group.api.mdx

```diff
@@ -0,0 +1,282 @@
+---
+id: add-application-to-group
+title: "Assign an application to a group (Work-in-Progress)"
+description: "Assigns an application to a group."
+sidebar_label: "Assign an application to a group (Work-in-Progress)"
+hide_title: true
+hide_table_of_contents: true
+api: eJztWE1v2zgQ/SsDnlqsYrlt2m11M9KPzaJbBKmzPSQ5jMSxzYYiVZKKYxj674sh5USx3W5R7GUBHwxbIjmf75H0W4uAcy+KS/HB2bYR15mwDTkMyppTKQqBUk6aRqsqvpraNC0TknzlVMMvRSEm3qu58YAG8GE2BAsIc14xujJXZrpQHiZnp6A8KAMIX6y7OVLm6MzZuSPvwQcMBGgkLJXW0BpJbm6hsYFMUKihdIQ3ysyhWqCZk4fWBKWvjAoeHGlCT7BUYcGRtE1la55bK2PdZnh0ZUQmGnRYUyDHua+FwZpEIWKop1JkQnFWDYaFyISjb61yJEURXEvbqU8XlFKE07cjkQlfLahGUaxFWDVs1AenzFx0XXbvZ1CjX/A2rPC/+bxmg76xxpPn8efj5/z1Y6NL9ICxoyTBt1VF3s9arVfc0LDJdyS6TByPx/vtNc7eKkkSJAbkhhsb4Ba1khxwZQ13lJcOHOeNs6Wm+revnu2sB3ltwQ3O0kyQFFBpsOVXqgKghzSxJMkIuzx/fwJvjl/+fv1kEULjizxfLpcjN6uOSKpg3ci6ee5mFX943tMRTBfkCGpcQUmAUir2iZoTasgFRR58Q5WaqWpTjj5s4OInePVtSGEx3O4XPzRpu1mZmFlXYxCFaJ3apRhcnJ+CkkyE2YphveM6rplhq9kGlrYNRanR3HCjggp6r9NtL76ta3QrsLM9DrpMMEXbQRpCmUBzcsP4lQkvnu/YZlj8MZ2eQTIBlZUEM+sg8L7QO+IkamVU3daiOB6PM1HjXXp6NR53bJM7/hOZGKC7RqNJmN5KRxmoraMePzExZXxAU/1XnbFOzdW23xFvA5teiB7Eb1NGXddFQr3Yxft760olJZkIT+AtgnzYsAq1tks68OrAqwOvfsCr4/0HVTq7450hHm3qlvhYjYcgs2tmW3Pg1oFbB279gFtvfuJSucMw5QG1I5Srh9vm4wvmgXQH0h1It590L/f985oY4Co7xiE5Zx3YqmqdIwnLhdLRPP+b2/jub5IJiweuHbh24Nou17pM1BQWluWwpo3IYammEHk8pny+7nWjLh/wxufrRzJPxzoNuduN5NQ6LQqxTmzqijxfL6wPXbFurAtdfsutukWnsNQJoDycWLdBk7YV6vh6X1d5gPWmTbYnWLdGIryG83efp/ABAy1xFavMLh+bfj1+Pd5rlad+xyIreynDhMnBPrExy4TfazZN/hnDUdPyVLVOhdVnXpbKUxI6cpOWO3OPld5ftM7PaZLI+h/vNwj688s0gkCZmY3LezDsBsJdIedT5OPRs13gnZ1G/lS2rlsT22/mvSY5SKzSrQ+cUCa0qsj4CPFeH9xM+5hG4O/kEZ6NuCkJOZu9c67Coi1Hla3zKi27/y61LfMalcl7Fz4/mfx18ent5Ojj6cm7T5/fHT0bjUfhLsTkG+tDjWYQR5J1v6/qwpNtBffpdj3WD8fK/08l7lEU6C7kjUZlGNex/Ot+D7hMmrEXmSge1OPhNsAjj/Xe66zn8qVYr0v0dOF01/Hrby25lSgurx+oH/cKqTz/lqKYofbbovCwxE/Oe/n4KWwL03uT6V+iWcXtRrf8JDJxQ6uBHs7S9S/HsCtX/0IkjyvYXXeZWBBKcrE+ac6kqqgJg9XfvcXwFnK/pZ9dMPf/Act+pLY=
+sidebar_class_name: "put api-method"
+info_path: docs/apis-tools/camunda-api-rest/specifications/camunda-8-rest-api
+custom_edit_url: null
+hide_send_button: true
+---
+
+import MethodEndpoint from "@theme/ApiExplorer/MethodEndpoint";
+import ParamsDetails from "@theme/ParamsDetails";
+import RequestSchema from "@theme/RequestSchema";
+import StatusCodes from "@theme/StatusCodes";
+import OperationTabs from "@theme/OperationTabs";
+import TabItem from "@theme/TabItem";
+import Heading from "@theme/Heading";
+
+<Heading
+  as={"h1"}
+  className={"openapi__heading"}
+  children={"Assign an application to a group (Work-in-Progress)"}
+></Heading>
+
+<MethodEndpoint
+  method={"put"}
+  path={"/groups/{groupId}/applications/{applicationId}"}
+  context={"endpoint"}
+></MethodEndpoint>
+
+Assigns an application to a group.
+
+This API is in a Work-in-Progress state and will undergo potential breaking changes until
```

Comment:
@tmetzke Do we have these breaking changes already documented in announcements?

Reply:
Those are more like breaking changes in 8.8 itself between alphas, nothing we need to document. It's more of a precaution warning because those endpoints are under heavy refactoring 😅

Reply:
See [internal thread](https://camunda.slack.com/archives/C06UYJMMETZ/p1741330563661189?thread_ts=1741295012.417169&cid=C06UYJMMETZ)

Reply:
I have [asked the Identity project](https://camunda.slack.com/archives/C06UYJMMETZ/p1746688981726729) to not add more such endpoints and remove the wording from those that are considered refactored now.

Reply:
Thanks for bringing this up! 🙏